### PR TITLE
Fix DateInlineRow, TimeInlineRow and DateTimeInlineRow style/rend…

### PR DESCRIPTION
**Pull request by kamerc**
Fix DateInlineRow, TimeInlineRow and DateTimeInlineRow style/rendering issues for iOS14. See screenshots from the Example app for details. The default picker view is .inline to match the iOS native apps. If a user is on iOS 13 or lower, they will still see the default style of wheels

### DateInlineRow (before)
dateBefore
### DateInlineRow (after)
dateAfter

### TimeInlineRow (before)
timeBefore
### TimeInlineRow (after)
timeAfter

### DateTimeInlineRow (before)
dateTimeBefore
### DateTimeInlineRow (after)
dateTimeAfter

### CODE REVIEW

**Comment by funkenstrahlen** [comment]
Thank you!

File: Source/Rows/DateInlineRow.swift [hunk 1]
```diff
@@ -105,7 +105,16 @@ open class _CountDownInlineRow: _DateInlineFieldRow {
 public final class DateInlineRow_<T>: _DateInlineRow, RowType, InlineRowType {
     required public init(tag: String?) {
         super.init(tag: tag)
-        onExpandInlineRow { cell, row, _ in
+        onExpandInlineRow { cell, row, inlineRow in
+            inlineRow.cellUpdate() { cell, row in
+                cell.datePicker.datePickerMode = .date
```
Comment (mats-claassen) [inline Source/Rows/DateInlineRow.swift:110]: I think setting datePickerMode is not needed here, right?

**Comment by mats-claassen** [comment]
I agree with the change but I think `onExpandInlineRow` isn't the best place for this as anyone who overrides this has to copy this code as well to maintain the functionality. From what I saw, if we remove the line setting the `datePickerMode` it is the same code for each date row except CountDownRow which does not support `.inline` mode. What about adding this change to `configureInlineRow` in `DatePickerRowProtocol`? You will have to check whether `datePickerMode` is countdown to avoid applying this change to CountDown rows

File: Source/Rows/DateInlineRow.swift (hunk 1 above)
Comment (kamerc) [reply Source/Rows/DateInlineRow.swift:110]: From the documentation: default is UIDatePickerModeDateAndTime. So this line needs to stay for DateInlineRow and also the line that sets it to .time on the TimeInlineRow. But it looks like the line for DateTimeInlineRow that sets it to .dateAndTime can be removed.

**Comment by kamerc** [comment]
@mats-claassen configureInlineRow takes DatePickerRowProtocol rather than DatePickerRow, which I need to be able to access the cell.datePicker. What if I add a new method to call from setupInlineRow that passes DatePickerRow?

**Comment by mats-claassen** [comment]
Sounds good. Then just don't call it in CountDown row

**Comment by kamerc** [comment]
@mats-claassen I have pushed an update

File: Source/Rows/DateInlineRow.swift (hunk 1 above)
Comment (mats-claassen) [reply Source/Rows/DateInlineRow.swift]: I mean `datePickerMode` is set in `DatePickerCell` (line 54). It is not necessary to re-set it here

**Pull request merged**